waiter/config-full.edn

```diff
@@ -350,6 +350,11 @@
                                 "scale-factor" 1
                                 "scale-up-factor" 0.1}
 
+ ;; Service description parameters with numeric values, e.g. cpus, can be configured to have
```

Comment:
Does this apply to all parameters with numeric values?

Reply:
Yes, I updated the schema in `settings.clj` to reflect this.